.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,220 @@
+        id: docker
+        uses: cafuego/command-output@main
+        with:
+          run: |
+            docker-compose -f tests/docker-compose.yml up -d
+            sleep 10
+            docker ps -a
+            docker-compose -f tests/docker-compose.yml exec -w /srv/www -T drupal composer self-update 2.5.5
+            docker-compose -f tests/docker-compose.yml exec -w /srv/www -T drupal composer install --dev
+        env:
+          fail-fast: true
+
+      - name: PHPCS
+        id: phpcs
+        uses: cafuego/command-output@main
+        with:
+          run: |
+            docker-compose -f tests/docker-compose.yml exec -w /srv/www -T drupal phpcs vendor/drupal/coder/coder_sniffer
+            docker-compose -f tests/docker-compose.yml exec -u appuser -w /srv/www -T drupal phpcs -np --report=full --standard=phpcs.xml ./html/sites/all/modules/custom ./html/sites/all/modules/access ./html/sites/all/modules/ar ./html/sites/all/modules/ev ./html/sites/all/modules/hdx ./html/sites/all/modules/hid ./html/sites/all/modules/hr
+        env:
+          fail-fast: true
+
+      - name: Find Comment
+        uses: peter-evans/find-comment@v2
+        id: fc
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          issue-number: ${{ github.event.pull_request.number }}
+          comment-author: 'github-actions[bot]'
+          body-includes: Build output
+
+      - name: Create or update comment
+        uses: peter-evans/create-or-update-comment@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          comment-id: ${{ steps.fc.outputs.comment-id }}
+          issue-number: ${{ github.event.pull_request.number }}
+          body: |
+            ### Test Run Output
+            #### Software Versions
+            #### Composer Validate `${{ steps.validate.outcome }}`
+            #### PHP Lint `${{ steps.lint.outcome }}`
+            #### Docker Build `${{ steps.build.outcome }}`
+            #### Environment Setup `${{ steps.docker.outcome }}`
+            #### Site Install `${{ steps.install.outcome }}`
+            #### PHP Code Sniffer `${{ steps.phpcs.outcome }}`
+
+            <details><summary>Software Versions</summary>
+            <code>${{ steps.versions.outputs.stdout }}</code>
+            <code>${{ steps.versions.outputs.stderr }}</code>
+            </details>
+
+            <details><summary>Composer Validate</summary>
+            <code>${{ steps.validate.outputs.stdout }}</code>
+            <code>${{ steps.validate.outputs.stderr }}</code>
+            </details>
+
+            <details><summary>PHP Lint</summary>
+            <code>${{ steps.lint.outputs.stdout }}</code>
+            <code>${{ steps.lint.outputs.stderr }}</code>
+            </details>
+
+            <details><summary>PHPCS</summary>
+            <code>${{ steps.phpcs.outputs.stdout }}</code>
+            <code>${{ steps.phpcs.outputs.stderr }}</code>
+            </details>
+
+            <details><summary>Tests</summary>
```

Comment:
Doesn't make much difference, but we could skip this and the Site Install if we aren't running those steps.